src/components/OpenSeadragonViewer.js

```diff
@@ -97,6 +123,20 @@ export class OpenSeadragonViewer extends Component {
     });
   }
 
+  /**
+   * annotationsToContext - converts anontations to a canvas context
+   */
+  annotationsToContext(annotations) {
```

Comment:
This is intended to be super basic, but hopefully can provide us an extension point to do more complex things. This likely will live elsewhere in the future.